packages/blocks-to-image/src/index.spec.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  Actions,
+  Blocks,
+  Button,
+  Divider,
+  Image,
+  MdSection,
+} from '@slack-wrench/blocks';
+import { toMatchImageSnapshot } from 'jest-image-snapshot';
+import { times } from 'ramda';
+
+import BlockKitRenderer from './index';
+
+expect.extend({ toMatchImageSnapshot });
+
+const Resturant = (description: string, image: string) =>
+  MdSection(description, {
+    accessory: Image(image, 'alt text for image'),
+  });
+
+const ResturantBlocks = (resturantTimes = 1) =>
```

Comment:
/s/Resturant/Restaurant/g

```suggestion
const RestaurantBlocks = (restaurantTimes = 1) =>
```